src/main/java/no/nav/tag/finnkandidatapi/kandidat/KandidatController.java

```diff
@@ -22,10 +27,28 @@
 
     private final KandidatService kandidatService;
     private final TilgangskontrollService tilgangskontroll;
+    // TODO: Fjern kafka ting
+    private final KafkaTemplate<String, String> kafkaTemplate;
 
     @GetMapping("/{fnr}")
     public ResponseEntity<Kandidat> hentKandidat(@PathVariable("fnr") String fnr) {
         loggBrukAvEndepunkt("hentKandidat");
+
+        // TODO: Fjern kafka ting
+        if (fnr.equals("123")) {
+            kafkaTemplate.send("en-kafka-topic", fnr, "jayso");
+        } else {
+            try {
+                OppfølgingAvsluttetMelding oppfølgingAvsluttetMelding = OppfølgingAvsluttetMelding.builder()
+                        .aktorId(fnr)
+                        .sluttdato(new Date()).build();
+                String melding = new ObjectMapper().writeValueAsString(oppfølgingAvsluttetMelding);
+                kafkaTemplate.send("en-kafka-topic", fnr, melding);
+            } catch (JsonProcessingException e) {
+                e.printStackTrace();
```

Comment:
Bør ikke bruke printstacktrace, den kan skrive til feil sted og på feil format, tilsvarende system.out.println. Vi bør bruke Log.error med e som andre argument i steden.

Reply:
Ja dette var bare for testing lokalt, har tatt bort all koden i controlleren :)